plugins/MantisGraph/MantisGraph.php

```diff
@@ -113,6 +113,7 @@ function resources() {
 			html_javascript_link( 'chart-' . CHARTJS_VERSION . '.min.js' );
 			html_javascript_link( 'chart.bundle-' . CHARTJS_VERSION . '.min.js' );
 		}
+		html_javascript_link( '../plugins/MantisGraph/MantisGraph.js' );
```

Comment:
Currently chart-xxxx.min.js files are in the js folder for core.  They should be moved into plugins/MantisGraph/files as well.